
<launch>
    <!-- vim: set ft=xml noet : -->
    <!-- Packet loss and rate testing launch file-->
    <!-- UAV0 -->
    <group ns="uav0">
        <!-- mavcomm param -->
	<arg name="serial_port" default="/dev/ttyUSB0" />
	<arg name="baudrate" 	default="115200" />
        <arg name="Flag_ats153" default="2" />
	<!-- mavcomm -->
	<node pkg="mavcomm" type="mavcomm" name="mavcomm" output="screen" respawn="false"> 
          <!-- common param -->          
          <param name="my_id"     	value="1" />
          <param name="Flag_1ShowRn"     value="0" />
          <!-- mavcomm param -->	  
          <param name="serial_port"     value="$(arg serial_port)" />
	  <param name="serial_baudrate" value="$(arg baudrate)" />
          <param name="Flag_ats153"     value="$(arg Flag_ats153)" />
        </node>
    </group>
    <group ns="uav1">
        <!-- mavcomm param -->
	<arg name="serial_port" default="/dev/ttyUSB1" />
	<arg name="baudrate" 	default="115200" />
        <arg name="Flag_ats153" default="2" />
	<!-- mavcomm -->
	<node pkg="mavcomm" type="mavcomm" name="mavcomm" output="screen" respawn="false"> 
          <!-- common param -->          
          <param name="my_id"     	value="5" />
          <param name="Flag_1ShowRn"     value="2" />
          <!-- mavcomm param -->	  
          <param name="serial_port"     value="$(arg serial_port)" />
	  <param name="serial_baudrate" value="$(arg baudrate)" />
          <param name="Flag_ats153"     value="$(arg Flag_ats153)" />
        </node>
    </group>
</launch>
